apps/studio/tsconfig.json

```diff
@@ -36,4 +36,4 @@
     "./../../packages/ui/src/**/*.d.ts"
   ],
   "exclude": ["node_modules"]
-}
```

Comment:
remove files from your PR, which doesn't contain any change. This helps in keeping the PR crisp